### PR TITLE
chore: remove global styles

### PR DESCRIPTION
**Pull request by bashirkarimi**
Removed: 
- root
- reboot
- bootstrap/type
- bootstrap/license

A separate ticket is created to fix the broken styles: https://github.com/orgs/swisspost/projects/3/views/12?pane=issue&itemId=80363901

### CODE REVIEW

File: packages/styles/src/components/base.scss
Comment (oliverschuerch) [inline packages/styles/src/components/base.scss:1]: Let's include the leftover of this file into the `elements/body.scss` file, so we can get rid of this file completely.

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/documentation/.storybook/styles/components/typo.scss [hunk 1]
```diff
@@ -24,7 +24,7 @@ body {
     }
 
     .docs-story > :first-child {
-      font-family: var(--bs-body-font-family);
+      font-family: var(--post-core-font-family-frutiger);
```
[the remark this answers is not in the document]
Comment (bashirkarimi) [reply packages/documentation/.storybook/styles/components/typo.scss:27]: The value of `--post-body-font-family` is `Swiss Post Sans v005`. Should we use `Swiss Post Sans v005` or `Frutiger Neue For Post`?

File: packages/styles/src/components/reboot.scss [hunk 2]
```diff
@@ -1,13 +0,0 @@
-@forward './../variables/options';
-
-@use './../lic/bootstrap-license';
-
-@use './../mixins/type';
```
[the remark this answers is not in the document]
Comment (bashirkarimi) [reply packages/styles/src/components/reboot.scss:5]: It is used in many files for font curves. It should be removed after the fonts curves are no longer is used.

File: packages/styles/src/components/type.scss
[the remark this answers is not in the document]
Comment (bashirkarimi) [reply packages/styles/src/components/type.scss:1]: It is removed.

File: packages/styles/src/themes/bootstrap/_forms.scss [hunk 3]
```diff
@@ -1,4 +1,3 @@
-@forward './forms/labels';
```
[the remark this answers is not in the document]
Comment (bashirkarimi) [reply packages/styles/src/themes/bootstrap/_forms.scss:1]: It is removed.

File: packages/styles/src/components/base.scss
Comment (bashirkarimi) [reply packages/styles/src/components/base.scss:1]: It is done.

File: packages/documentation/.storybook/styles/components/typo.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (bashirkarimi) [reply packages/documentation/.storybook/styles/components/typo.scss]: I have  implemented your suggestion and will check the correct font-family in the https://github.com/orgs/swisspost/projects/3/views/17?pane=issue&itemId=78602494

File: packages/documentation/.storybook/styles/components/typo.scss [hunk 4]
```diff
@@ -38,6 +38,6 @@ body {
 
   // full-screen preview
   #storybook-root {
-    font-family: var(--bs-body-font-family);
+    font-family: var(--post-core-font-family-frutiger);
```
[the remark this answers is not in the document]
Comment (bashirkarimi) [reply packages/documentation/.storybook/styles/components/typo.scss]: It is implemented.

File: packages/documentation/.storybook/styles/components/typo.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (oliverschuerch) [reply packages/documentation/.storybook/styles/components/typo.scss]: Let's use the token ``--post-body-font-family` as is and rename the font name in the `@font-face` definition in the `font.scss` file to match the new name.

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/documentation/.storybook/styles/components/typo.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (oliverschuerch) [reply packages/documentation/.storybook/styles/components/typo.scss]: The fonts.scss file should be adapted like this:

```scss
@forward './../variables/options';

@use './../variables/options';

@font-face {
  font-display: swap;
  font-family: 'Swiss Post Sans v005';
  font-weight: 900;
  src:
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-Blk.woff2') format('woff2'),
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-Blk.woff') format('woff');
}

@font-face {
  font-display: swap;
  font-family: 'Swiss Post Sans v005';
  font-weight: 900;
  font-style: italic;
  src:
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-BlkIt.woff2') format('woff2'),
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-BlkIt.woff') format('woff');
}

@font-face {
  font-display: swap;
  font-family: 'Swiss Post Sans v005';
  font-weight: 700;
  src:
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-Bold.woff2') format('woff2'),
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-Bold.woff') format('woff');
}

@font-face {
  font-display: swap;
  font-family: 'Swiss Post Sans v005';
  font-weight: 700;
  font-style: italic;
  src:
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-BdIt.woff2') format('woff2'),
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-BdIt.woff') format('woff');
}

@font-face {
  font-display: swap;
  font-family: 'Swiss Post Sans v005';
  font-weight: 400;
  src:
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-Rg.woff2') format('woff2'),
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-Rg.woff') format('woff');
}

@font-face {
  font-display: swap;
  font-family: 'Swiss Post Sans v005';
  font-weight: 400;
  font-style: italic;
  src:
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-It.woff2') format('woff2'),
    url('#{options.$font-base-path}/FrutigerNeueforPostW05-It.woff') format('woff');
}

```

[the remark this answers is not in the document]
Comment (bashirkarimi) [reply packages/documentation/.storybook/styles/components/typo.scss]: The fonts are renamed.

File: packages/styles/src/elements/body.scss [hunk 5]
```diff
@@ -1,13 +1,11 @@
...
+@use '../components/fonts';
+// fixes for color in buttons
 @use './../mixins/color' as mx-color;
-@use './../variables/animation';
 @use './../variables/color';
 
 body {
-  @include type.font-smoothing();
   @include mx-color.colored-background(color.$white);
 }
```
[the remark this answers is not in the document]
Comment (bashirkarimi) [reply packages/styles/src/elements/body.scss]: It is removed.

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**